framework/src/governance/lib/data-catalog-database.ts

```diff
@@ -47,10 +54,58 @@ export class DataCatalogDatabase extends TrackedConstruct {
    * KMS encryption Key used by the Crawler
    */
   readonly crawlerLogEncryptionKey?: IKey;
+  /**
+   * The DataLakeSettings for Lake Formation
+   */
+  readonly dataLakeSettings?: CfnDataLakeSettings;
+  /**
+   * The IAM Role used by Lake Formation to access data.
+   */
+  readonly lfDataAccessRole?: IRole;
+  /**
+   * The Lake Formation data lake location
+   */
+  readonly dataLakeLocation?: CfnResource;
+  /**
+   * The custom resource for revoking IAM permissions from the database
+   */
+  readonly revokeIamAllowedPrincipal?: AwsCustomResource;
+  /**
+   * The Lake Formation grant on the database for the Crawler when Lake Formation or Hybrid is used
+   */
+  readonly crawlerLfDbGrant?: CfnPrincipalPermissions;
+  /**
+   * The Lake Formation grant on the tables for the Crawler when Lake Formation or Hybrid is used
+   */
+  readonly crawlerLfTablesGrant?: CfnPrincipalPermissions;
+  /**
+   * The Lake Formation grant on the data location for the Crawler when Lake Formation or Hybrid is used
+   */
+  readonly crawlerLfLocationGrant?: CfnPrincipalPermissions;
+  /**
+   * The IAM Role used to revoke LakeFormation IAMAllowedPrincipals
+   */
+  readonly lfRevokeRole?: IRole;
+  /**
+   * The Lake Formation grant on the data location for the CDK role
+   */
+  readonly cdkLfLocationGrant?: CfnPrincipalPermissions;
```

Comment:
Can you clarify this is only available when using lakeformation, is it also the case for hybrid?

Reply:
not needed anymore